ILRepack.Tests/ILRepack.Tests.csproj

```diff
@@ -55,6 +58,15 @@
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="sharpdevelop\ICSharpCode.SharpDevelop.BuildWorker120.exe.config" />
```

Comment:
:+1: I was thinking about grabbing dlls from nuget, as it's *the* repository these days